src/providers/test/get-transaction-count.test.ts

```diff
@@ -1,47 +1,55 @@
-import { ethers } from 'ethers';
-import Web3 from 'web3';
-import { jsonRpcProvider } from '../..';
+import * as unfetch from 'isomorphic-unfetch';
+import { jsonRpcProvider, tinyBig } from '../..';
 import type { BlockTag } from '../../types/Block.types';
+import { mockOf } from './mock-of';
 import { rpcUrls } from './rpc-urls';
+import { buildFetchInit, buildRPCPostBody } from '../../classes/utils/fetchers';
+
+jest.mock('isomorphic-unfetch');
 
-// coinbase 1 hotwallet
```

Comment:
I removed the comment above your hot wallet address because now that we are mocking the address we use is arbitrary